### DISCUSSION TITLE
Pleased to see Exaile's still here!

### DISCUSSION THREAD

**Discussion by Sidnioulz** [exaile/exaile#905] [General]
Hey folks,

Just popping in to say hi and kudos on staying an active FOSS project for so long! I used to contribute back in Launchpad some 13-14 years ago (under nickname `sidi`) and it throws me back to see Exaile is still up and running.

I hope there will be many more decades to come :)

**Comment by virtuald** [comment]
Me too! As long as GTK/GStreamer keep their APIs stable...
